tests/unit/APITest.ts

```diff
@@ -14,16 +15,26 @@ import * as TestHelper from '../utils/TestHelper';
 import waitForBatchedUpdates from '../utils/waitForBatchedUpdates';
 import waitForNetworkPromises from '../utils/waitForNetworkPromises';
 
+const Onyx = reactNativeOnyxMock;
```

Comment:
```suggestion
const Onyx = MockedOnyx as ReactNativeOnyxMock;
```

Use it here and that should be enough